src/job-template.Dockerfile

```diff
@@ -8,6 +9,18 @@ ENV PROXY_MODE "rewrite"
 {% if manifest.wrapper_properties and manifest.wrapper_properties['user_module_port'] %}
 ENV JOB_USER_MODULE_PORT "{{ manifest.wrapper_properties['user_module_port'] }}"
 {% endif %}
+{% endif %}
+
+{% if manifest.jobtype_extra %}
```

Comment:
In this case I think we can surely just rename `wrapper_properties` to `jobtype_extra`, not caring about backward compatibility. This job type is not widely used. Plus, it's all fine, if we switch to next major version of this plugin: `version: 3.0.0` in a `plugin-manifest.yaml`. Plugins are immutable so old job type will still work with a new Racetrack core.